features/GEM052_Correct-geometric-subcontexts.feature

```diff
@@ -0,0 +1,20 @@
+@implementer-agreement
+@GEM
+@version1
+@E00050
+Feature: GEM052 - Correct geometric subcontexts
+The rule verifies that there is a minimum of at least one subcontext per context. The context
```

Comment:
```suggestion
The rule verifies that there is a minimum of at least one subcontext per context. 
Reference: https://github.com/buildingSMART/Sample-Test-Files/issues/137.
The context
```

A reference to this discussion would be helpful.  As you noted in the JIRA ticket for GEM050 there is nothing documented in the spec for this.

Which begs the question - shouldn't the requirement for subcontext be a best practice warning and not a validation fail?